packages/terra-text/src/terra-dev-site/doc/text/Text.1.doc.mdx

```diff
@@ -0,0 +1,64 @@
+import { Badge } from 'terra-text/package.json?dev-site-package';
+
+import TextColors from '../example/TextColors?dev-site-example';
+import TextDisplays from '../example/TextDisplays?dev-site-example';
+import TextFontSizes from '../example/TextFontSizes?dev-site-example';
+import TextVariations from '../example/TextVariations?dev-site-example';
+import TextVisuallyHidden from '../example/TextVisuallyHidden?dev-site-example';
+import TextWeights from '../example/TextWeights?dev-site-example';
+import TextWordWrapped from '../example/TextWordWrapped?dev-site-example';
+
+import TextPropsTable from 'terra-text/src/Text?dev-site-props-table';
+
+<Badge />
+
+# Terra Text
+
+The font size, font weight, font family, and font color in terra components are set to defaults in terra-base which is then inherited into all components. Components can override these base styles as needed in their specific component CSS. In some cases, you may need text that differs from the base font styles, though, the text doesn't belong to a specific component.
+
+In these cases, the text component may be helpful. Using terra-text, you can create text that differs from the base text styles by changing the font size, font weight, and color.
+
```

Comment:
We should add a section describing how to use color correctly

Reply:
Do we need first-hand documentation for color? 
As that is available through our examples.

Reply:
IMO the docs in the examples are confusing so we should have first-hand docs

Reply:
Do we want to display CSS file to users for what CSS is being passed to the example?
Because other than showing what is in `attention` or other classes, the example seems pretty clear.

Reply:
I think we need to add documentation in addition to the examples. I think the examples as is are fine and clear, if you have a background with css modules.

Reply:
updated here https://github.com/cerner/terra-core/pull/2886/commits/0d42d80f668800e87b89a6cf78c2ccf9c666f730.